i18n/config.py

```diff
@@ -9,13 +9,13 @@
 
 # BASE_DIR is the working directory to execute django-admin commands from.
 # Typically this should be the 'edx-platform' directory.
-BASE_DIR = Path('.').abspath()  # pylint: disable=invalid-name
+BASE_DIR = Path('.').abspath()
 
 # The base filename for the configuration file.
 BASE_CONFIG_FILENAME = 'config.yaml'
 
 
-class Configuration(object):
+class Configuration:
```

Comment:
This should keep the explicit `object` superclass until we've dropped Python 2.7 support.